src/Microsoft.AspNetCore.Mvc.Razor/Internal/CompositeTagHelperDescriptorResolver.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Razor.Compilation.TagHelpers;
+using Microsoft.AspNetCore.Razor.Runtime.TagHelpers;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.Internal
+{
+    public class CompositeTagHelperDescriptorResolver : ITagHelperDescriptorResolver
+    {
+        public IList<TagHelperDescriptorResolver> _resolvers;
+
+        public CompositeTagHelperDescriptorResolver(
```

Comment:
Spoke offline. Since the `CompositeTagHelperDescriptorResolver` is in `.internal` namespace, we don't need to provide extensibility for it.